simulator/src/main/java/com/github/benmanes/caffeine/cache/simulator/policy/product/CaffeinePolicy.java

```diff
@@ -67,8 +67,10 @@ public void record(AccessEvent event) {
     if (value == null) {
       cache.put(event.key(), event);
       policyStats.recordMiss();
+      policyStats.recordWeightedMiss(event.weight());
```

Comment:
Would it make sense to have `recordWeightedMiss` also perform `recordMiss`, so implementations use one or the other? 

Otherwise I think these would be named `recordHitWeight` and `recordMissWeight` to indicate that they maintain the weights and not also the hit/miss counts.

Reply:
Good point. I'll combine them.